src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,78 @@
 package core.basesyntax;
 
 public class SalaryInfo {
+    static final int INDEX_START_DAY_DATE = 0;
+    static final int INDEX_END_DAY_DATE = 2;
+    static final int INDEX_START_MONTH_DATE = 3;
+    static final int INDEX_END_MONTH_DATE = 5;
+    static final int INDEX_START_YEAR_DATE = 6;
+    static final int INDEX_END_YEAR_DATE = 10;
+
+    int parseInt(int index1, int index2, String s) {
+        return Integer.parseInt(s.substring(index1, index2));
```

Comment:
You need to use LocalDate with Formatter here instead of returning int for every single part of the date.